app/views/recipes/_template_field.html.erb

```diff
@@ -0,0 +1,21 @@
+<div data-controller="dynamic-form-field visibility"
```

Comment:
The template field uses the dynamic-form-field controller to replace the provisional `index` coming from the server with the epoch as suggested by the rails guides.

This partial and the ingredient field are very similar in terms of structure and style but I kept them separated to make the example a bit simpler. I'm sure there must be a way to have a single template but it'll be come a bit cumbersome.